content/en/providers/check-in/sp/_index.md

```diff
@@ -419,6 +498,7 @@ Check-in supports the following OpenID Connect/OAuth2 grant types:
   support of delegated access to resources.
 - Device Code: used by devices that lack a browser to perform a user-agent based
   OAuth flow.
+- Client Credentials: used by services to execute periodic tasks
```

Comment:
Client credentials: used by clients to obtain an access token outside of the context of a user. Such an access token is typically used by clients to access resources about themselves rather than to access a user's resources.

Reply:
done